### PR TITLE
feat(stickers): Add CSS vars for $sticker-

### PR DESCRIPTION
**Pull request by isabellechanclou**
Add CSS vars in Stickers component and Docs.

This is part of https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1196.

Preview link: https://deploy-preview-1251--boosted.netlify.app/

### CODE REVIEW

**Review by louismaximepiton — CHANGES_REQUESTED** [review]
⚠️ The sticker component is broken ⚠️

File: scss/_sticker.scss [hunk 1]
```diff
@@ -1,39 +1,43 @@
...
-  font-weight: $font-weight-bold;
+  width: var(--#{$prefix}sticker-size);
+  height: var(--#{$prefix}sticker-size);
+  font-weight: var(--#{$prefix}sticker-font-weight);
   text-align: center;
   word-wrap: break-word;
-  background-color: $brand-orange;
-  border-radius: $sticker-size-lg; // stylelint-disable-line property-disallowed-list
+  background-color: var(--#{$prefix}sticker-background-color);
+  // border-radius: $sticker-size-lg; // stylelint-disable-line property-disallowed-list
```
[the remark this answers is not in the document]
Comment (isabellechanclou) [reply scss/_sticker.scss]: Sure. It was supposed to be removed.

File: scss/_sticker.scss [hunk 2]
```diff
@@ -1,39 +1,43 @@
...
+  border-radius: var(#{$prefix}sticker-size=#{$sticker-size-lg}); // stylelint-disable-line property-disallowed-list
```
[the remark this answers is not in the document]
Comment (isabellechanclou) [reply scss/_sticker.scss]: Oups, I committed thinking I had finished writing the value but in fact not!

**Review by louismaximepiton — APPROVED** [review]
LGTM 🚀

File: site/content/docs/5.1/components/sticker.md [hunk 3]
```diff
@@ -83,8 +83,22 @@ Since stickers only provide a container, accessibility becomes specific to the s
...
+
+{{< scss-docs name="sticker-css-vars" file="scss/_sticker.scss" >}}
+
+`sm` and `lg` modifier classes are used to update the value of these CSS variables as needed for **small** and **large** stickers:
+
+{{< scss-docs name="sticker-sm-css-vars" file="scss/_sticker.scss" >}}
+
+{{< scss-docs name="sticker-lg-css-vars" file="scss/_sticker.scss" >}}
+
+### Sass variables
```
Comment (julien-deramond) [inline site/content/docs/5.1/components/sticker.md:102]: Just for info @isabellechanclou and @louismaximepiton in the other (or next) PRs regarding CSS custom properties, don't forget to use the new doc structure (see https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1251/commits/8623bdbb3d1eb09eff5362d4cb004f74b8d83cc9).

Comment (louismaximepiton) [reply site/content/docs/5.1/components/sticker.md:102]: Ah yup, didn't saw it, Thanks ! 😄

**Review by julien-deramond — APPROVED** [review]

**Pull request merged**